homeassistant/components/foscam/__init__.py

```diff
@@ -81,3 +99,45 @@ def update_unique_id(entry):
     LOGGER.info("Migration to version %s successful", entry.version)
 
     return True
+
+
+class FoscamCoordinator(DataUpdateCoordinator):
+    """My custom coordinator."""
+
+    def __init__(
+        self,
+        hass: HomeAssistant,
+        session: FoscamCamera,
+        update_interval=timedelta(seconds=30),
+    ) -> None:
+        """Initialize my coordinator."""
+        super().__init__(
+            hass,
+            LOGGER,
+            # Name of the data. For logging purposes.
+            name=DOMAIN,
+            # Polling interval. Will only be polled if there are subscribers.
+            update_interval=update_interval,
```

Comment:
```suggestion
            update_interval=timedelta(seconds=30),
```

Reply:
This is propagating the argument, I would like to keep it this way